website/docs/r/network_manager_routing_rule.html.markdown

```diff
@@ -0,0 +1,126 @@
+---
+subcategory: "Network"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_network_manager_routing_rule"
+description: |-
+  Manages a Network Manager Routing Rule.
+---
+
+
+# azurerm_network_manager_routing_rule
+
+Manages a Network Manager Routing Rule.
+
+!> **Note:** Terraform has enabled force deletion. This setting deletes the resource even if it's part of a deployed configuration. If the configuration is deployed, the service will perform a cleanup deployment in the background before the deletion.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resources"
+  location = "West Europe"
+}
+
+data "azurerm_subscription" "current" {}
+
+resource "azurerm_network_manager" "example" {
+  name                = "example-network-manager"
+  resource_group_name = azurerm_resource_group.example.name
+  location            = azurerm_resource_group.example.location
+  scope {
+    subscription_ids = [data.azurerm_subscription.current.id]
+  }
+  scope_accesses = ["Routing"]
+}
+
+resource "azurerm_network_manager_network_group" "example" {
+  name               = "example-network-group"
+  network_manager_id = azurerm_network_manager.example.id
+}
+
+resource "azurerm_network_manager_routing_configuration" "example" {
+  name               = "example-routing-configuration"
+  network_manager_id = azurerm_network_manager.example.id
+}
+
+resource "azurerm_network_manager_routing_rule_collection" "example" {
+  name                     = "example-routing-rule-collection"
+  routing_configuration_id = azurerm_network_manager_routing_configuration.example.id
+  network_group_ids        = [azurerm_network_manager_network_group.example.id]
+  description              = "example routing rule collection"
+}
+
+resource "azurerm_network_manager_routing_rule" "example" {
+  name               = "example-routing-rule"
+  rule_collection_id = azurerm_network_manager_routing_rule_collection.example.id
+  description        = "example routing rule"
+
+  destination {
+    type    = "AddressPrefix"
+    address = "10.0.0.0/16"
```

Comment:
Shouldn't this be 10.0.0.0/24?

```suggestion
    address = "10.0.0.0/24"
```

[Product Documentation](https://learn.microsoft.com/azure/virtual-network-manager/concept-user-defined-route#common-destination-patterns-for-ip-addresses):
```markdown
For traffic destined to the private space through a network virtual appliance,
enter `192.168.0.0/16`, `172.16.0.0/12`, `40.0.0.0/24`, `10.0.0.0/24` as the destination
in the rule. These destinations are based on the `RFC1918` private IP address space.
```

Reply:
updated